core/state_processor.go

```diff
@@ -95,6 +95,18 @@ func ApplyTransaction(config *params.ChainConfig, bc ChainContext, author *commo
 	// Create a new environment which holds all relevant information
 	// about the transaction and calling mechanisms.
 	vmenv := vm.NewEVM(context, statedb, config, cfg)
+
+	if config.IsYoloV2(header.Number) {
+		statedb.AddAccessListAccount(msg.From())
+		if dst := msg.To(); dst != nil {
+			statedb.AddAccessListAccount(*dst)
+			// If it's a create-tx, the destination will be added inside evm.create
+		}
+		for _, addr := range vmenv.ActivePrecompiles() {
+			statedb.AddAccessListAccount(addr)
```

Comment:
Out of curiosity, do we have a test for ripemd? :) Does that precompile exist on the chain currently or not?

Reply:
RipeMD most certainly exist